### ISSUE TITLE
install for Mac os and M1 apple chip

### ISSUE THREAD

**Issue by liyongkang123** [rusty1s/pytorch_geometric#2736]
Hi, I want to install this library in my M1 macbook air, and Now I have install Python 3.8 and Pytorch-cpu 1.8 , But I can't find an ARM64 version of this library that supports MacOS.  Only macosx_10_9_x86_64.whl are found, 
Will Apple computers with M1 chip be supported in the future?

**Comment by rusty1s** [comment]
Yes, I plan to upport M1 as well, I just haven't had time to do this yet. In the meantime, you may try to install from source without any binaries.

**Comment by shi27feng** [comment]
@liyongkang123 follow this link https://github.com/rusty1s/pytorch_scatter/issues/183
the installation from source is easy:
MACOSX_DEPLOYMENT_TARGET=11.1 CC=clang CXX=clang++ python3 setup.py install

**Comment by rusty1s** [comment]
I just checked and it seems that GitHub Actions do not support building on M1 yet. There's not much I can do here besides waiting for GitHub to support it, sorry :(

**Comment by RiegelGestr** [comment]
hope to help I ve installed all via conda (assuming that in your env `torch` has been yet installed):
`pip install torch-scatter torch-sparse torch-geometric`
and all my previous code work fine. wp
